packages/core/README.md

```diff
@@ -1,340 +1,77 @@
-You can read more about this [here](https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation)
+To run these steps manually refer to [this section of our guide](https://embrace.io/docs/react-native/integration/add-embrace-sdk/#manually)
 
-### Step 5: Install latest pods
+## Initialize the Embrace SDK
 
-Once you're done linking, it's necessary to download the pods that were linked to the project.
-To do so, go into the iOS project folder inside of your React Native app directory, where the Podfile should be located, and run:
+Initialize method applies the necessary listener to your application. This allow Embrace to track javascript errors, check js bundle changes (if you use OTA), track js patch and react native versions.
 
-```console
-$ pod install
-```
-
-This should download `RNEmbrace` pod to your iOS Project.
+```javascript
+import React, {useEffect, useState} from 'react'
+import {initialize} from '@embrace-io/react-native';
 
-### Step 6: Call Methods
+const App = ()=> {
 
-A good place to start is after your startup moment ends. This moment starts automatically with the code added to AppDelegate.m on iOS or MainActivity.java on Android, and you should end on Component mount similar to the below code.
+  useEffect(()=>{
+    // Note: Initialize is a promise, so if you want to perform an action and it must be tracked, it is recommended to use await to wait for the method to finish
 
-```javascript
-import { endAppStartup } from "embrace-io";
+    initialize().then(hasStarted=>{
+      if(hasStarted){
+         //doSomething
+      }
+    });
+  },[])
 
-type Props = {};
-export default class App extends Component<Props> {
-  componentDidMount() {
-    endAppStartup();
-  }
+ return ...
 }
+export default App
 ```
 
-### Step 7: Integrate the User Identifier, Logs and Moments into your App
-
-Since React Native is built with either an iOS or Android native framework, most of the functionality integrated to effectively use Embrace can be called either in Swift / Obj-C, Java, or Javascript. Please follow the iOS Quick Integration or Android Quick Integration for the remaining steps to complete your integration.
-
-_The following calls located in the file, embrace.js, are listed below for reference._
-
-```javascript
-// Startup: Call each place a startup may conclude.  Remember those deeplinks!
-endAppStartup(properties: ?Dictionary<Any>)
-
-// User Identifiers: Set (or clear) one or more User Identifiers on app start up or registration
-setUserIdentifier(userIdentifier: String)
-clearUserIdentifier()
-
-setUsername (username: String)
-clearUsername()
-
-setUserEmail(userEmail: String)
-clearUserEmail()
+## Uploading sourcemaps
 
-setUserAsPayer()
-clearUserAsPayer()
-// Breadcrumbs: Log a Breadcrumb to display these on the User Timelines
-addBreadcrumb(message: String)
+Refer to our guide on [uploading symbol files](https://embrace.io/docs/react-native/integration/upload-symbol-files/).
```

Comment:
Would add a sentence before for context:
> The Embrace SDK allows you to view both native and JavaScript stack traces for crashes and error logs.

Reply:
updated